test/unittests/client/remote_manager_test.py

```diff
@@ -19,7 +36,47 @@ def test_compress_files(self):
             "Two_file.txt": os.path.join(folder, "Two_file.txt"),
         }
 
-        path = compress_files(files, PACKAGE_TGZ_NAME, dest_dir=folder)
+        path = compress_files(files, "conan_package.txz", dest_dir=folder, compressformat="xz")
         assert os.path.exists(path)
-        expected_path = os.path.join(folder, PACKAGE_TGZ_NAME)
+        expected_path = os.path.join(folder, "conan_package.txz")
         assert path == expected_path
+
+        extract_dir = os.path.join(folder, "extracted")
+        uncompress_file(path, extract_dir)
+
+        extract_files = list(sorted(os.listdir(extract_dir)))
+        expected_files = sorted(files.keys())
+        assert extract_files == expected_files
+
+        for name, path in files.items():
+            extract_path = os.path.join(extract_dir, name)
+            with open(path, "r") as f1, open(extract_path, "r") as f2:
+                assert f1.read() == f2.read()
+
+    @pytest.mark.skipif(sys.version_info.minor < 14, reason="zstd needs Python >= 3.14")
+    def test_compress_and_uncompress_zst_files(self):
```

Comment:
Should we add a test for the error when you configure zst in a python < 3.14?
I have checked and it raises this error so it's fine: `Error while compressing: unknown compression type 'zst'`